### ISSUE TITLE
formGroup property missing within the CellCloseEvent class definition

### ISSUE THREAD

**Issue by Horizon0156** [telerik/kendo-angular#1704]
## I'm submitting a...

* Bug report
* Documentation issue

## Current behavior
I'm missing the `formGroup` property within the `CellCloseEvent` class definition. While using an anonymous type, I'm able to access the form group within the passed event argument.

## Expected behavior
Please add the form group to the event's class definition so the event handler can expect a typed argument.

## Minimal reproduction of the problem with instructions
From HTML template:
`<kendo-grid (cellClose)="handleCellClosed($event)">`

From corresponding component:
```
public handleCellClosed(args: CellCloseEvent) {
    if (args.formGroup.valid) {
         // save changes
    }
}
```
Does not complile (error TS2339: Property 'formGroup' does not exist on type 'CellCloseEvent')

While using anonymous type works without any problems.
```
public handleCellClosed(args: any) {
    if (args.formGroup.valid) {
         // save changes
    }
}
```
https://www.telerik.com/kendo-angular-ui/components/grid/editing/in-cell-editing/

## Environment

Package versions:

```
"@progress/kendo-angular-grid": "3.5.1",
```

Browser:
Doesn't matter

System:
- TypeScript version: 2.7.2
- Node version: 8.11.2  
- Platform: Windows

**Comment by danielkaradachki** [comment]
Thanks for pointing this out. The field is available in the latest [dev](https://www.telerik.com/kendo-angular-ui-develop/components/installation/development-builds/) version.

**Comment by Horizon0156** [comment]
Great, thanks a lot. Shall I close this issue, or do you need it to be open for regression / tracking / release purposes?

**Comment by danielkaradachki** [comment]
We prefer to leave the issues opened until there is an official version with the fix.

**Comment by danielkaradachki** [comment]
fixed in 3.6.1
